static/css/main.css

```diff
@@ -372,26 +378,79 @@ button:hover,
   backface-visibility: hidden;
 }
 
-@media screen and (max-width:700px) {
+/* Responsive */
+
+@media screen and (max-width: 620px) {
+  .mobile-nav {
+    display: flex;
+    padding: 1rem 2rem;
+  }
+  .mobile-nav::after {
+    content: "\f0c9";
+    font-family: fontAwesome;
+    margin-left: 1rem;
+    width: 1rem;
+  }
+  .mobile-nav:hover {
+    background: none;
+    color: #00bf6b;
+  }
+  .mobile-nav.is-open::after {
+    content: "\f00d";
+  }
+  body > header nav.mobile-close {
+    display: none;
+  }
+  body > header nav {
+    position: absolute;
+    top: 6rem;
+    left: 0;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-color: white;
+    padding: 3rem;
+    box-shadow: 1px 1px 2px #eee;
+  }
+  body > header nav a {
+    font-size: 2.25rem;
+    line-height: 5rem;
+  }
+  
+  nav .avatar {
+    display: none;
+    background: none;
+    border: none;
+  }
+  nav .profile {
+    opacity: 1;
+    position: relative;
+    height: 100%;
+    width: 100%;
+    box-shadow: none;
+    border: none;
+    top: 0;
+    right: 0;
+  }
+  nav .profile::before {
+    display: none;
+  }
+  nav .profile > li {
+    display: flex;
+    justify-content: center;
+  }
+  nav .profile > li > a{
+    padding: 3px 15px;
```

Comment:
Any specific reason to use `px` here vs `rem`?

Reply:
because nobody like css :)